gc/gc-base/src/main/java/org/projectnessie/versioned/persist/gc/IcebergExpiredContentValues.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.versioned.persist.gc;
+
+import java.util.HashSet;
+import java.util.Set;
+import org.projectnessie.model.Content;
+import org.projectnessie.model.IcebergTable;
+
+public final class IcebergExpiredContentValues extends ExpiredContentValues {
```

Comment:
we still have good separation.

For validation, if the snapshot id's are there instead of the content object I thought it will easy. so plugged in iceberg content. 
I can remove it. But writing test cases and validation logic becomes hard